spec/index.bs

```diff
@@ -1103,6 +1106,11 @@ When a {{GPURenderPassEncoder}} is created, it has the following default state:
       * `x, y` = `0, 0`
       * `width, height` = the dimensions of the pass's render targets
 
+When a {{GPURenderBundle}} is executed, it does not inherit the pass's pipeline,
```

Comment:
That's unfortunate, allowing some state to be inherited would be useful to build more modular render bundles, especially inheriting bind groups for the bind group that contains scene-global data. That said going with this more restrictive approach first sounds good.

Reply:
Do we know what the 3 different APIs do?

Reply:
@litherum I think #382 covered it well, with one exception

In Vulkan secondary command buffer can be executed in the middle of a renderpass so it at least inherits the renderpass state (by necessity).